com.unity.testing.graphics-performance/Documentation~/Grafana-database-scheme.md

```diff
@@ -0,0 +1,100 @@
+| PlayerSettings. AndroidMinimumSdkVersion          | STRING    | NULLABLE |
+| PlayerSettings. AndroidTargetSdkVersion           | STRING    | NULLABLE |
+| PlayerSettings. AndroidSdkVersion                 | STRING    | NULLABLE |
+| PlayerSettings. ScriptingRuntimeVersion           | STRING    | NULLABLE |
+| PlayerSettings. EnabledXrTargets                  | STRING    | NULLABLE |
+| ProjectVersions                                   | RECORD    | REPEATED |
+| ProjectVersions. ProjectName                      | STRING    | NULLABLE |
+| ProjectVersions. Changeset                        | STRING    | NULLABLE |
+| ProjectVersions. Branch                           | STRING    | NULLABLE |
+| ProjectVersions. Date                             | TIMESTAMP | NULLABLE |
+| Results                                           | RECORD    | REPEATED |
+| Results. TestName                                 | STRING    | NULLABLE |
+| Results. TestCategories                           | RECORD    | REPEATED |
+| Results.TestCategories. Name                      | STRING    | NULLABLE |
+| Results. TestVersion                              | STRING    | NULLABLE |
+| Results. StartTime                                | STRING    | NULLABLE |
+| Results. EndTime                                  | STRING    | NULLABLE |
+| Results. SampleGroups                             | RECORD    | REPEATED |
+| Results.SampleGroups. Samples                     | RECORD    | REPEATED |
+| Results.SampleGroups.Samples. Value               | FLOAT     | NULLABLE |
+| Results.SampleGroups. AggregatedSampleValue       | FLOAT     | NULLABLE |
+| Results.SampleGroups. Min                         | FLOAT     | NULLABLE |
+| Results.SampleGroups. Max                         | FLOAT     | NULLABLE |
+| Results.SampleGroups. Median                      | FLOAT     | NULLABLE |
+| Results.SampleGroups. Average                     | FLOAT     | NULLABLE |
+| Results.SampleGroups. StandardDeviation           | FLOAT     | NULLABLE |
+| Results.SampleGroups. PercentileValue             | FLOAT     | NULLABLE |
+| Results.SampleGroups. Sum                         | FLOAT     | NULLABLE |
+| Results.SampleGroups. Zeroes                      | INTEGER   | NULLABLE |
+| Results.SampleGroups. SampleCount                 | INTEGER   | NULLABLE |
+| Results.SampleGroups. Definition                  | RECORD    | NULLABLE |
+| Results.SampleGroups.Definition. Name             | STRING    | NULLABLE |
+| Results.SampleGroups.Definition. SampleUnit       | STRING    | NULLABLE |
+| Results.SampleGroups.Definition. AggregationType  | STRING    | NULLABLE |
+| Results.SampleGroups.Definition. Threshold        | FLOAT     | NULLABLE |
+| Results.SampleGroups.Definition. IncreaseIsBetter | BOOLEAN   | NULLABLE |
+| Results.SampleGroups.Definition. Percentile       | FLOAT     | NULLABLE |
+| TestProject                                       | STRING    | NULLABLE |
```

Comment:
Those fields have been added to the db:

```
JobMetaData RECORD  NULLABLE    
JobMetaData. Bokken RECORD  NULLABLE    
JobMetaData.Bokken. ResourceGroupName   STRING  NULLABLE    
JobMetaData.Bokken. DeviceResourceId    STRING  NULLABLE    
JobMetaData.Bokken. DeviceId    STRING  NULLABLE    
JobMetaData.Bokken. DeviceIp    STRING  NULLABLE    
JobMetaData.Bokken. DeviceType  STRING  NULLABLE    
JobMetaData.Bokken. HostIp  STRING  NULLABLE    
JobMetaData.Bokken. RuntimeOs   STRING  NULLABLE    
JobMetaData. Yamato RECORD  NULLABLE    
JobMetaData.Yamato. JobFriendlyName STRING  NULLABLE    
JobMetaData.Yamato. JobName STRING  NULLABLE    
JobMetaData.Yamato. JobId   STRING  NULLABLE    
JobMetaData.Yamato. ProjectId   STRING  NULLABLE    
JobMetaData.Yamato. ProjectName STRING  NULLABLE    
JobMetaData.Yamato. WorkDir STRING  NULLABLE    
JobMetaData.Yamato. JobOwnerEmail   STRING  NULLABLE    
Dependencies    RECORD  NULLABLE    
Dependencies. Packages  STRING  REPEATED    
```

Can you add them to the table?
Thanks!

Reply:
Done, thankyou!